src/aliceVision/sensorDB/cameraSensors.db

```diff
@@ -378,6 +378,7 @@ Blackberry;Blackberry Priv;0.28;devicespecifications
 Blackberry;Blackberry Q10;4.8;devicespecifications
 Blackberry;Blackberry Q5;4.8;devicespecifications
 Blackmagic;Blackmagic Studio Camera HD;12.48;usercontribution
+Blackmagic;Blackmagic Pocket Cinema Camera 4K;23.10;usercontribution
```

Comment:
isn't 18.96?
https://www.blackmagicdesign.com/products/blackmagicpocketcinemacamera/techspecs/W-CIN-12

Reply:
Ooops 23.10 seems to be for the 6K version!